pytorch_to_returnn/torch/nn/modules/variable.py

```diff
@@ -26,11 +26,12 @@ def make_output_tensor_from_returnn(self, inputs_flat: List[Tensor], layer: Laye
     naming = Naming.get_instance()
     values = None
     if naming.import_params_from_torch_namespace and self.parent_mod:
-      parent_mod, param_name = self.parent_mod
-      mod_abs_name = naming.get_module_abs_id_name(parent_mod)
-      torch_mod = naming.import_params_from_torch_namespace.get_module_by_abs_id_name(mod_abs_name)
-      torch_param = getattr(torch_mod, param_name)
-      values = torch_param.detach().numpy()
+      if not ".tmp_root" in layer.network.name:  # temp layer
```

Comment:
You always should use `if ... not in ...` (PyCharm should tell you, I think).

But I also think there should be a better way to check for a temp layer. I'm not sure you really should look at `layer.network.name` here.

Reply:
We could do `if naming.tmp_eager_root_namespace.name not in layer.network.name:` in order not to hard-code `".tmp_root"`. For `layer.network.name` I currently don't know an alternative. I'm open to suggestions of course. Might also be a separate issue?

Reply:
I just checked the code now. It's also wrong in `apply_call`. It's wrong because the meaning of `network.name` changed (it was never really meant to represent anything except for debugging).
We should use `layer.get_absolute_name()` (both in `apply_call` and here). I just pushed that for `apply_call` in 306557fe17b00ea3acfd4.

Reply:
Ah yes, good!